Hush/Assets/Scripts/Enemies/EnemySword.cs

```diff
@@ -9,8 +8,8 @@ public class EnemySword : MonoBehaviour, IWeapon
     {
         #region Parameters
 
-        [Header("References")]
-        [SerializeField] private Enemy enemy;
+        [Header("References")] [SerializeField]
+        private Enemy enemy;
```

Comment:
```suggestion
        [Header("References")]
        [SerializeField] private Enemy enemy;
```

Reply:
You seem to be very opinionated abt formatting. 
Do you have a specific linter/style rules you want to enforce for this project?

Reply:
I use the default rules set by Unity in Rider! Don't know if ReSharper would apply the same linting on VS if that's what you use and don't think there's an equivalent on VSCode...

Reply:
Do you have a particular linter set up that you want the team/style to use? If not, do you wanna make one? Could make it easier than having to leave all these formatting comments :P

Reply:
I use the default Rider one..... Interesting

Reply:
I think this is very close to what I follow...
https://www.gamedeveloper.com/disciplines/coding-guideline-for-unity-c-

Don't think it's worth implementing a true linter for whatever IDE the team uses though 😸

Reply:
Change made in 530f00d